### DISCUSSION TITLE
Linear strain-softening -Mohr Coulomb

### DISCUSSION THREAD

**Discussion by rksin8** [idaholab/moose#19042] [Q&A Modules: Solid mechanics] [answered]
How can I implement linear strain-softening to Mohr-Coulomb failure criterion? 

![image](https://user-images.githubusercontent.com/22324848/136519180-d21f1705-3aa8-4750-bc15-478445127e5e.png)

**Comment by hugary1995** [comment, marked as answer]
You can take a look at this example: `moose/modules/tensor_mechanics/test/tests/mohr_coulomb/uni_axial2.i`

**Reply by rksin8** [reply]
Thank you for sharing this. Found a similar example using TensorMechanicsHardeningExponential.
